tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -228,42 +256,60 @@ func GetMSPIDForOrg(connProfilePath, orgName string) (string, error) {
 	yamlFile, err := ioutil.ReadFile(connProfilePath)
 	if err != nil {
 		logger.ERROR("Failed to read connectionprofile to get MSPID ")
-		return mspID, err
+		return config, err
 	}
 	err = yaml.Unmarshal(yamlFile, &config)
 	if err != nil {
-		logger.ERROR("Failed to create GetMSPID object")
-		return mspID, err
+		logger.ERROR("Failed to create GetConnProfileOptions object")
+		return config, err
 	}
-	mspID = config.Organizations[orgName].MSPID
-	return mspID, nil
+	return config, nil
 }
 
-//queryInstalledusingCLI -- querying installed cc using cli
-func (i InstantiateCCUIObject) queryInstalledusingCLI(instantiateObject InstantiateCCUIObject) (InstalledCC, error) {
+func fetchOrdererInformation(currentDir string) ([]string, error) {
 
-	queryInstalled := InstalledCC{}
-	orgName := instantiateObject.ChannelOpt.OrgName[0]
-	currentDir, err := paths.GetCurrentDir()
+	var ordererOrgPath string
+	var ordererDirectoryName string
+	var ordererName []string
+	files, err := ioutil.ReadDir(fmt.Sprintf("%s/crypto-config/ordererOrganizations/", currentDir))
 	if err != nil {
-		return queryInstalled, err
+		logger.ERROR("failed opening directory")
+		return ordererName, err
 	}
-	err = SetEnvForCLI(orgName, instantiateObject.ConnProfilePath, instantiateObject.TLS, currentDir)
+	for _, file := range files {
+		ordererOrgPath = file.Name()
+		break
+	}
+	ordererDirectory, err := ioutil.ReadDir(fmt.Sprintf("%s/crypto-config/ordererOrganizations/%s/orderers/", currentDir, ordererOrgPath))
 	if err != nil {
-		return queryInstalled, err
+		logger.ERROR("failed opening directory")
+		return ordererName, err
+	}
+	for _, file := range ordererDirectory {
+		ordererDirectoryName = file.Name()
+		break
 	}
+	ordererName = strings.Split(ordererDirectoryName, ".")
```

Comment:
This becomes `ordererName := strings.Split(ordererDirectoryName, ".")` by removing the `var ordererName` above